### DISCUSSION TITLE
Local Variables guide

### DISCUSSION THREAD

**Discussion by CodionCA** [File-New-Project/EarTrumpet#1548] [Q&A] [answered]
Is there any info on the different local variables available to use on the dev build? I see the example used is "IsInGameMode" but I cannot find any documentation on this anywhere.

Thanks!

**Comment by riverar** [comment, marked as answer]
@CodionCA Hey there, sorry for the confusion. That was just an example. Variables are meant for you to define and use, perhaps in other actions.

**Reply by CodionCA** [reply]
Oh I understand, yes I was confused I thought this was a windows function, my bad. I see now they can be defined through other actions, thanks!
